src/ui/Select/Select.tsx

```diff
@@ -73,7 +82,7 @@ const Select: React.FC<ISelect> = ({
                   onClick={() => handleOnChange(language.value)}
                   selected={selectedOption.id === language.id}
                 >
-                  {language.id.toUpperCase()} - {language.name}
+                  {language.name}
```

Comment:
maybe you can change the name of the singular element of the array to option
language => option 
could be better in my opinion